lib/teleterm/clusters/cluster_file_transfer.go

```diff
@@ -54,6 +55,9 @@ func (c *Cluster) TransferFile(ctx context.Context, request *api.FileTransferReq
 
 	err = AddMetadataToRetryableError(ctx, func() error {
 		err := c.clusterClient.TransferFiles(ctx, request.GetLogin(), serverUUID+":0", config)
+		if errors.As(err, new(*sftp.NonRecursiveDirectoryTransferError)) {
+			return trace.Errorf("transferring directories through Teleport Connect is not supported at the moment, please use tsh scp -r")
```

Comment:
Is there value in showing which path was a directory and thus wasn't individually copied?

Reply:
Both Web UI and Connect show the message next to the path that was transferred, so I don't think it's worth to duplicate it in the error message.